google-api-client/src/test/java/com/google/api/client/googleapis/json/GoogleJsonResponseExceptionTest.java

```diff
@@ -55,7 +56,13 @@ public void testFrom_withDetails() throws Exception {
         GoogleJsonResponseException.from(GoogleJsonErrorTest.FACTORY, response);
     assertEquals(GoogleJsonErrorTest.ERROR, GoogleJsonErrorTest.FACTORY.toString(ge.getDetails()));
     assertTrue(
-        ge.getMessage(), ge.getMessage().startsWith("403" + StringUtils.LINE_SEPARATOR + "{"));
+        ge.getMessage(),
```

Comment:
ge should probably be unabbreviated. Reading it here, I have no idea what this variable is.

Reply:
Done.